### PR TITLE
add incremental export feature

### PR DESCRIPTION
**Pull request by bvarent**
Adds support for [incremental exports](https://developer.zendesk.com/rest_api/docs/core/incremental_export) for tickets, users and organizations. Which is something different from the current /export implemented for Tickets.

also see issue #58

### CODE REVIEW

File: tests/Zendesk/API/LiveTests/TicketsTest.php [hunk 1]
```diff
@@ -170,6 +170,13 @@ public function testExport() {
         $this->assertEquals($this->client->getDebug()->lastResponseCode, '200', 'Does not return HTTP code 200');
     }
 
+    public function testIncremental() {
+        $tickets = $this->client->tickets()->export(array('start_time' => '1332034771'));
```
Comment (shanitang) [inline tests/Zendesk/API/LiveTests/TicketsTest.php]: Is this test for `incremental ` or `export` ?

**Comment by shanitang** [comment]
@bvarent  Thanks for adding this!! Could you please add unit tests as well, because we only run unit tests for Travis CI?

**Comment by buheryfi** [comment]
@bvarent, I've noticed that you've added the Incremental endpoints as a method to the class for users, organizations, and tickets, rather than creating a separate class for Incremental.  Our documentation shows it as a separate class and our client wrapper also implements separate classes in general, so it would be best to add a separate Incremental Class for these endpoints rather than including a method in existing classes.  Thoughts?

**Comment by bvarent** [comment]
@buheryfi: Agreed. I think I can find time somewhere this week to turn it into a separate endpoint class.

**Comment by atroche** [comment]
@bvarent let us know if you're still keen on doing this. I'll close the PR for now.

**Pull request closed without merging**